### PR TITLE
Update facts endpoint to support multiple network sources (tolerate vcenter but ignore)

### LINKED ISSUE #329: Update facts endpoint to support multiple network sources (tolerate vcenter but ignore)
# Opening a new issue
___

## Specify type:
- Enhancement

### Bug severity (if applicable):
- High

___

## Description:
Update facts endpoint to support multiple network sources (tolerate vcenter but ignore)

/facts/:
```
{
    "sources": [
        {
            "source":"id",
            "source_type":"network",
            "facts":[
                
            ]
        }
    ]
}
```

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #329 
  
Code coverage will decrease on this because there is some code that needs to be refactored as part of #348.  I will tackle test coverage in #348.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ build: clean-cli
 	$(PYTHON) setup.py build -f
 
 clean-cli:
-	-rm -rf dist/ build/ quipucords.egg-info/
+	-rm -rf dist/ build/ quipucords.egg-info/ quipucords/raw_facts/
```
Comment (chambridge) [inline Makefile]: Seems unrelated to the cli

File: quipucords/api/fact/raw_fact_util.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2017 Red Hat, Inc.
```
Comment (chambridge) [inline quipucords/api/fact/raw_fact_util.py]: old copyright

File: quipucords/api/signals/fact_collection_receiver.py [hunk 3]
```diff
@@ -12,37 +12,29 @@
...
+from api.serializers import FingerprintSerializer
 
 ENGINE = Engine()
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
-@receiver(post_save, sender=FactCollection)
 def process_fact_collection(sender, instance, **kwargs):
-    """Process facts using engine and convert to fingerprints.
+    """Restart a scan.
```
Comment (chambridge) [inline quipucords/api/signals/fact_collection_receiver.py]: Feels like a bad comment change.

File: quipucords/api/tests_report.py [hunk 4]
```diff
@@ -10,95 +10,94 @@
...
-        url = '/api/v1/reports/'
-
-        # Create a system fingerprint via collection receiver
-        self.create_fact_collection(
-            os_versions=['7.4', '7.4', '7.5'])
-
-        # Query API
-        response = self.client.get(url, {'fact_collection_id': 2})
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+# import uuid
```
Comment (chambridge) [inline quipucords/api/tests_report.py]: Is there an issue to re-implement these tests?

Comment (kholdaway) [reply quipucords/api/tests_report.py]: I plan to clean these up in #348

File: quipucords/api/signals/fact_collection_receiver.py (hunk 3 above)
Comment (kholdaway) [reply quipucords/api/signals/fact_collection_receiver.py]: Fixed with #348

File: quipucords/api/fact/raw_fact_util.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/api/fact/raw_fact_util.py]: Fixed with #348

File: Makefile (hunk 1 above)
Comment (kholdaway) [reply Makefile]: Fixed with #348
